### PR TITLE
System wide installation for Windows

### PR DESCRIPTION
**Pull request by ntaraujo**
Implements #9

I strongly recommend do not use it in future. Windows is kinda dumb and needs a better approach for running this safely. Maybe PyInstaller with InnoSetup.

The installation method presupposes `python` is available in `%PATH%`

### CODE REVIEW

File: install.py [hunk 1]
```diff
@@ -8,12 +8,25 @@
 with open('photon.py') as w:
     code = w.read()
 
-code = code.replace('@PHOTON_INSTALL_PATH',f'"{os.getcwd()}/"')
+code = code.replace('@PHOTON_INSTALL_PATH', f'r"{os.getcwd()}/"')
```
Comment (ntaraujo) [inline install.py:11]: Forgot to mention, the use of a normal string here instead of a raw one was causing a SyntaxError (unicode error)

**Comment by HashLDash** [comment]
do jeito que esta, o comando fica disponivel no cmd como photon?

**Comment by ntaraujo** [comment]
Sim, `photon.bat` é chamado como `photon`. O Windows não lê shebangs como `#!/usr/bin/env python3`, por isso o segundo arquivo.

**Comment by HashLDash** [comment]
Boa

**Pull request merged**